synchronizer/synchronizer.go

```diff
@@ -407,18 +451,23 @@ func (s *ClientSynchronizer) Sync() error {
 				waitDuration = s.cfg.SyncInterval.Duration
 			}
 			//Sync L1Blocks
+			resetDone, lastEthBlockSynced, err := s.checkReorgAndExecuteReset(lastEthBlockSynced)
+			if resetDone || err != nil {
+				continue
+			}
+
 			startL1 := time.Now()
 			if s.l1SyncOrchestration != nil && (latestSyncedBatch < latestSequencedBatchNumber || !s.cfg.L1ParallelSynchronization.FallbackToSequentialModeOnSynchronized) {
 				log.Infof("Syncing L1 blocks in parallel lastEthBlockSynced=%d", lastEthBlockSynced.BlockNumber)
-				lastEthBlockSynced, err = s.syncBlocksParallel(lastEthBlockSynced)
+				_, err = s.syncBlocksParallel(lastEthBlockSynced)
```

Comment:
Why this change? I think we need that variable

Reply:
you are right, I got an error of lint `SA4006: this value of `lastEthBlockSynced` is never used (staticcheck)` but I was because a previous error!